lib/components/regions_map/components/serach_places/view_models/search_places_bloc.dart

```diff
@@ -31,6 +33,18 @@ class SearchPlacesBloc extends Bloc<SearchPlacesEvent, SearchPlacesState> {
   Future<void> _onQueryTextChange(OnQueryTextChange event, Emitter<SearchPlacesState> emit) async {
     if (event.query.isNotEmpty) {
       emit(state.copyWith(pageState: PageState.success, showLinearIndicator: true));
+
+      final List<PredictionModel> regionMatches = [];
+      // If regions not empty search them by matches and aggregates
+      if (state.regions.isNotEmpty) {
+        for (final i in state.regions) {
```

Comment:
i -> region

Reply:
I usually `i` for everiting if nested loops or callback `e`.
Do we really need the change here to clarify the intended code?